app/forms/course_years_form.rb

```diff
@@ -9,25 +9,45 @@ class CourseYearsForm
 
   validates :course_year, inclusion: { in: ->(rec) { rec.course_years_options.keys } }
 
-  def initialize(params = {})
+  def initialize(trainee: nil, params: {})
     assign_attributes(params)
+    @trainee = trainee
   end
 
   def course_year
     @course_year.to_i
   end
 
+  # { 2022=>"2022 to 2023", 2021 => "2021 to 2022" }
   def course_years_options
+    course_year_to.downto(course_year_from).map.index_with { |year| "#{year} to #{year + 1}" }
+  end
+
+private
+
+  def course_year_from
     [
-      default_course_year,
-      default_course_year - 1,
-    ].inject({}) do |sum, y|
-      sum[y] = "#{y} to #{y + 1}"
-      sum
-    end
+      recruitment_cycle_year,
+      itt_start_date_year,
+      course_year_to - 1,
+    ].compact.min
+  end
+
+  def itt_start_date_year
+    return unless trainee&.itt_start_date
+
+    AcademicCycle.for_date(trainee.itt_start_date).start_year
   end
 
-  def default_course_year
+  def recruitment_cycle_year
+    return unless trainee&.current_course&.recruitment_cycle_year
+
+    trainee.current_course.recruitment_cycle_year + 1
+  end
```

Comment:
I'm wary of using variables mentioning `recruitment_cycle` - these frequently get confused with academic cycles and end up with an off by one error. And sometimes they are the *same* as the academic cycle (eg they are poorly labelled).